### ISSUE TITLE
`first-run.py` can’t handle ssh repository URLs

### ISSUE THREAD

**Issue by khaledhosny** [googlefonts/Unified-Font-Repository#33]
It requires HTTPS URLs, so `git@github.com:foo/bar.git` URL will result in:
```
$ make build
. venv/bin/activate; python3 scripts/first-run.py
My git repository URL (git@github.com:foo/bar.git
) didn't look what I expected - are you hosting this on github?
You will need to do the initialization steps manually.
Read scripts/first-run.py for more instructions how to do this.
make: *** [Makefile:25: .init.stamp] Error 1
```

But I need SSH URL to be able to push to the repository without being asked about user name and password each time.

**Comment by eliheuer** [comment]
Same here, I use SSH for most font repos I work on. I'll try to fix this if @simoncozens is busy.

**Comment by eliheuer** [comment]
Simon told me this was just a simple regex issue, looking into a PR to fix this now...

**Comment by eliheuer** [comment]
This should be fixed as of commit 1aa99ac, please repon this issue if anyone has any problems with this still. Thanks @khaledhosny!
